src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.tsx

```diff
@@ -952,7 +965,9 @@ const ADT3DSceneBuilder: React.FC<IADT3DSceneBuilderCardProps> = (props) => {
     return (
         <DeeplinkContextProvider>
             <SceneThemeContextProvider>
-                <ADT3DSceneBuilderBase {...props} />
+                <SceneViewContextProvider outlinedMeshItems={[]}>
```

Comment:
does it  need to take in the outlinedMeshItems initial state if it's always empty array?